config.reek

```diff
@@ -2,5 +2,7 @@ IrresponsibleModule:
   enabled: false
 
 FeatureEnvy:
-  exclude:
```

Comment:
trc thì FeatureEnvy sẽ chạy trên toàn bộ project trừ cái file đc liệt kê trong exclude. Nhưng sau này thì sẽ có nhiều chỗ phải dùng biến ở class khác nên em tắt luôn nó đi